Sources/BraintreeApplePay/BTApplePayClient.m

```diff
@@ -24,6 +24,7 @@
 #endif
 
 NSString *const BTApplePayErrorDomain = @"com.braintreepayments.BTApplePayErrorDomain";
+NSInteger const NetworkConnectionLostCode = -1005;
```

Comment:
🧶 : is there a central place we could put this constant to make it more DRY?

Reply:
Yep, we can define this as a macro in `BTAPIClient` for now which is the preferred way to do this with how it's being used. This will also avoid the linker warnings we are getting that are making the SPM and Cocoapods builds fail.